tests/test_ref.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+import pytest
+
+from mlflow.ref import REF
+
+
+@pytest.mark.skipif("CI" not in os.environ, reason="Only works in CI")
+def test_ref():
+    assert REF is not None
```

Comment:
This test doesn't pass on local because `mlflow` is installed by `pip install -e .` for development.